Common.mk

```diff
@@ -410,17 +438,21 @@ validate-checksums: $(BINARY_TARGETS)
 	$(BUILDCTL)
 
 # Build helm chart
+ifeq ($(HAS_HELM_CHART),true)
 .PHONY: helm/build
-
+helm/build: ## Build helm chart
 helm/build: $(OUTPUT_DIR)/ATTRIBUTION.txt
-	$(BUILD_LIB)/helm_build.sh $(IMAGE_REPO) $(IMAGE_COMPONENT) $(IMAGE_TAG) $(OUTPUT_DIR)
+helm/build: $(if $(REPO_NO_CLONE),,$(HELM_GIT_CHECKOUT_TARGET))
```

Comment:
I know its ugly, but i think to be most correct we need to make sure this equals true, with the whole filter statement we do everywhere else... i think there is def room for improvement there, but probably best to stick with it until we change it more widely?